dbms/tests/queries/0_stateless/00918_json_functions.sql

```diff
@@ -11,6 +11,11 @@ SELECT JSONHas('{"a": "hello", "b": [-100, 200.0, 300]}', 'a');
 SELECT JSONHas('{"a": "hello", "b": [-100, 200.0, 300]}', 'b');
 SELECT JSONHas('{"a": "hello", "b": [-100, 200.0, 300]}', 'c');
 
+SELECT '--isValidJSON--';
+SELECT isValidJSON('{"a": "hello", "b": [-100, 200.0, 300]}');
+SELECT isValidJSON('not a json');
+SELECT isValidJSON('"HX-=');
```

Comment:
It seems like `isValidJSON('json')` works exactly as `JSONHas('json')`. So maybe it would be the better solution not to add another function, but add an alias

```
factory.registerAlias("isValidJSON", "JSONHas");
```

Reply:
Then you'll be able to pass additional argument to isValidJSON. I don't know if it's appropriate.

Reply:
It seems it's all about the name:
the `JSONIsValid` name can be considered as the function checking not only a passed json, but passed indices too (`JSONIsValid('json' [, indices])`);
the `IsValidJSON()` name looks more like it's just a check for a passed json only (`IsValidJSON('json')`. 
But we already have the function `JSONHas()`, so let's keep adding `IsValidJSON()` with a single `'json'` argument in this PR.